.github/workflows/production_actions.yml

```
@@ -91,10 +91,14 @@ jobs:
     - name: Restore NuGet Packages
       run: nuget restore Revit_glTF_Exporter.sln
     
-    - name: Setting new version to the App.config file
+    - name: Setting new version to the config file
       run: |
-        (Get-Content Common_glTF_Exporter/App.config) -replace 'key="version" value="[^"]+"', "key=`"version`" value=`"$env:FULL_VERSION`"" | Out-File -encoding ASCII Common_glTF_Exporter/App.config
+        (Get-Content Common_glTF_Exporter/Utils/SettingsConfig.cs) -replace '0.0.0', $env:FULL_VERSION | Out-File -encoding ASCII Common_glTF_Exporter/Utils/SettingsConfig.cs
     
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)